hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/MemberStateImpl.java

```diff
@@ -302,6 +312,8 @@ public JsonObject toJson() {
         serializeAsMap(root, "reliableTopicStats", reliableTopicsWithStats);
         serializeAsMap(root, "pnCounterStats", pnCountersWithStats);
         serializeAsMap(root, "executorStats", executorsWithStats);
+        serializeAsMap(root, "scheduledExecutorsWithStats", scheduledExecutorsWithStats);
```

Comment:
Minor: I see none of the other names of the keys have `With` in them.

Reply:
removed `with`